### PR TITLE
Added a robotic syringe

### LINKED ISSUE #19817: [Bug]: borg syringe can give people infections if used more than once
### Checks

- [x] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Horizon

### Describe the issue

using the syringe that the borg gets (Not the hypospray with preloaded chems) more than once gives people sepsis if used again, like a normal syringe.

This syringe is not disposable, as it it attached as one of your modules. You cannot sterilize it (as far as I'm aware) either, meaning it's essentially currently a one time use syringe.

### How to reproduce

1 - load syringe with chems and inject person 1

2 - load syringe with chems and inject persons 2 3 and 4

3 - wait for infections and or sepsis to happen and kill said persons after awhile.

4 - :surprisepikachu:

### Round ID

_No response_

### Staff ckey

_No response_

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Added a robotic syringe, to be used by borgs, that do not get dirty and gives infection to people after the first use.

Fixes #19817

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!bugfix

**Comment by FluffyGhoster** [comment]
!review

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 1]
```diff
@@ -381,6 +384,20 @@
...
+#############*/
+
+/**
+ * #Robotic syringe
+ *
+ * This syringe is for borgs and the likes, do not give it around otherwise
+ */
+/obj/item/reagent_containers/syringe/robotic
+	name = "robotic syringe"
+	desc = "A syringe for our syntetic friends."
```
Comment (alsoandanswer) [inline code/modules/reagents/reagent_containers/syringes.dm:398]:
```suggestion
	desc = "A syringe for our synthetic friends."
```

**Review by alsoandanswer — DISMISSED** [review]
The syringe should just be changed to a hypospray. No need for a new variable or code.

**Comment by FluffyGhoster** [comment]
> The syringe should just be changed to a hypospray. No need for a new variable or code.

Wouldn't solve the problem, they need syringes for blood content analysis

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**